index.html

```diff
@@ -200,46 +219,59 @@ <h2 class="about-me">
         <button class="get-my-resume">
             Get My Resume
         </button>    
```

Comment:
As pointed out by the first reviewer, please fix this overlapping of the button and the text in order to give your page visitors a good UI.

**See here**

<img width="637" alt="unverified" src="https://user-images.githubusercontent.com/84930355/206859927-3fed39ad-048e-4dc7-b3a4-d1689991bc82.png">